src/components/Projects/index.tsx

```diff
@@ -231,7 +231,7 @@ export default function Projects() {
           {concatenatedPages && (
             <Grid
               children={concatenatedPages.map((p, i) => (
-                <ProjectCard key={i} project={p} />
+                <ProjectCard key={i} project={p as ProjectCardProject} />
```

Comment:
Just trying to get `eslint`, curious how this got through CI.

Reply:
Wasn't sure if this is something I should get to the root of, or just get TSC to not complain.